content/rs/administering/troubleshooting/cluster-recovery.md

```diff
@@ -40,6 +40,9 @@ The cluster recovery process includes:
 - We recommend that you use clean persistent storage drives for the new cluster.
     If you use the original storage drives,
     make sure that you backup the files on the original storage drives to a safe location.
+- The cluster configuration file that you want to use as the configuration for the cluster.
+    The cluster configuration file is `/css/ccs-redis.rdb` on the persistent storage for each node.
+    Copy the cluster configuration file from one of the nodes to the persistent storage that you want to connect to the first node.
```

Comment:
As said before- this sentence is inaccurate. No need to copy anything, and the file is not located on the nodes...